src/components/AccountModule/ClientConnectionInfo.js

```diff
@@ -0,0 +1,161 @@
+    listening && online ? connectionStatus : CONNECTION_STATUS_ERROR
+  )
+
+  const Icon =
+    connectionStatus === CONNECTION_STATUS_HEALTHY ? IconCheck : IconCross
+  let connectionMessage = getConnectionMessage(
+    connectionStatus,
+    listening,
+    online,
+    clientNetworkName
+  )
+
+  if (connectionStatus === CONNECTION_STATUS_HEALTHY) {
+    connectionMessage = `Connected to Ethereum ${clientNetworkName} Network`
+  }
+
+  return (
+    <section
+      css={`
+        max-width: ${51 * GU}px;
+      `}
+    >
+      <h1
+        css={`
+          display: flex;
+          align-items: center;
+          height: ${4 * GU}px;
+          padding: 0 ${2 * GU}px;
+          ${textStyle('label2')};
+          color: ${theme.surfaceContentSecondary};
+          border-bottom: 1px solid ${theme.border};
+        `}
+      >
+        Ethereum connection
+      </h1>
+
+      <div
+        css={`
+          padding: ${2 * GU}px;
+        `}
+      >
+        <FlexWrapper
+          css={`
+            display: flex;
+            margin-top: ${1 * GU}px;
+            color: ${connectionStatusColor};
+            ${textStyle('label2')};
+          `}
+        >
+          <Icon size="small" />
+          {clientNetworkName && (
+            <span
+              css={`
+                margin-left: ${0.5 * GU}px;
+              `}
+            >
+              {connectionMessage}
+            </span>
+          )}
+        </FlexWrapper>
+        <ClientSyncedInfo
+          syncDelay={syncDelay}
+          listening={listening}
+          online={online}
+        />
+        <ConnectionInfoMessage
+          locator={locator}
+          connectionStatus={connectionStatus}
+        />
+      </div>
+    </section>
+  )
+}
+
+ClientConnectionInfo.propTypes = {
+  connectionStatus: PropTypes.symbol,
+  listening: PropTypes.bool,
+  locator: PropTypes.object,
+  online: PropTypes.bool,
+  syncDelay: PropTypes.number,
+}
+
+function ConnectionInfoMessage({ connectionStatus, locator }) {
+  let content = null
+  const handleNetworkSettingsClick = useCallback(() => {
+    window.location.hash = getAppPath({
+      dao: locator.dao || '',
+      search: getPreferencesSearch('network'),
+    })
+  }, [locator])
+
+  if (connectionStatus === CONNECTION_STATUS_WARNING) {
+    content = (
+      <span>
+        We've detected the Ethereum node you are connected to seems to be having
+        troubles syncing blocks. You can change the node settings in{' '}
+        <Link onClick={handleNetworkSettingsClick}>Network Settings.</Link>
+      </span>
+    )
+  }
+
+  if (connectionStatus === CONNECTION_STATUS_ERROR) {
+    content = (
+      <span>
+        We cannot connect to the Ethereum node. You can change the node settings
+        in
+        <Link onClick={handleNetworkSettingsClick}>Network Settings.</Link>
+      </span>
+    )
+  }
+
+  return (
+    <div
+      css={`
+        margin-top: ${GU}px;
+        margin-bottom: ${GU}px;
```

Comment:
As noted elsewhere, let's use `1 * GU` :).

For further context, this also makes it easier for us to replace in the future with a babel plugin (see https://github.com/aragon/aragon-ui/issues/333).